modules/monitoring-specifying-how-a-service-is-monitored-for-cluster-observability-operator.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// monitoring/cluster-observability-operator/configuring-the-cluster-observability-operator-to-monitor-a-service.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="specifying-how-a-service-is-monitored-for-cluster-observability-operator_{context}"]
+= Specifying how a service is monitored for Cluster Observability Operator
+
+To use the metrics exposed by your sample service, you must configure monitoring components to scrape metrics from the `/metrics` endpoint. 
```

Comment:
As before, I'd provide some context here about the sample service by referring to the other procedure maybe

Reply:
Done